README.md

```diff
@@ -70,6 +70,8 @@ Thus `https://[gitlab.domain/org/dependabot-script-repo]/pipeline_schedules` das
 * Set in the schedule the required variables:
   * `PROJECT_PATH`: `group/repository`
   * `PACKAGE_MANAGER_SET`: `bundler,composer,npm_and_yarn`
+* If you'd like to specify the directory that contains the package list in the repository, you can set the following environment variable:
```

Comment:
```suggestion
* If you'd like to specify the directory that contains the manifest file in the repository, you can set the following environment variable:
```

Reply:
Thank you. I've changed the wording as suggested.